src/ChameleonTemplate.php

```diff
@@ -43,13 +43,18 @@ class ChameleonTemplate extends BaseTemplate {
 	 * @throws \MWException
 	 */
 	public function execute() {
+		$bodyOnly = $this->getSkin()->getOptions()['bodyOnly'];
```

Comment:
In order to avoid `getOptions()` not existing, perhaps this can just be:
```php
$bodyOnly = version_compare( $this->configuration['wgVersion'], '1.39', '>=' );
```

Reply:
Fixed in force push.